visualization/include/pcl/visualization/impl/pcl_visualizer.hpp

```diff
@@ -669,7 +669,8 @@ pcl::visualization::PCLVisualizer::addText3D (
   else
     tid = id;
 
-  if (contains (tid))
+  std::string idToCheck = viewport > 0 ? tid.append(viewport, '*') : tid;
```

Comment:
Please call variables with `snake_case`. Also we need a space between `append` and bracket.

Reply:
addressed